rj_gameplay/rj_gameplay/play/basic122.py

```diff
@@ -20,9 +21,13 @@ def __init__(self):
         self.striker_tactic = striker_tactic.LineKickStrikerTactic(
             target_point=self.target_point)
         self.goalie_tactic = goalie_tactic.GoalieTactic()
-        self.wall_tactic = wall_tactic.WallTactic(2,
-                                                  role.Priority.LOW,
-                                                  cost_scale=0.1)
+        self.wall_tactic_1 = wall_tactic.WallTactic(role.Priority.LOW,
+                                                    cost_scale=0.1)
```

Comment:
for further discussion: is cost_scale needed here? (and if so, do we need role.Priority.LOW)